debug/src/check-props.js

```diff
@@ -0,0 +1,28 @@
+
+if (process.env.NODE_ENV !== 'production') {
+  var ReactPropTypesSecret = 'SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED',
```

Comment:
Let's use newer keywords like `const` or `let` here :+1:

Reply:
Yep, without the NODE_ENV stuff we don't need `var`. Done in bd70cf3 with a bit of judgment call (`ReactPropTypesSecret` definitely const, but I left the `loggedTypeFailures` as a `let` even though technically the reference itself is `const`…)

Reply:
That's perfectly fine we have lots of places where a `let` declaration could be a `const` :slightly_smiling_face: